DocSum/docker_image_build/build.yaml

```diff
@@ -11,18 +11,57 @@ services:
       context: ../
       dockerfile: ./Dockerfile
     image: ${REGISTRY:-opea}/docsum:${TAG:-latest}
+
   docsum-ui:
     build:
-      context: ../ui
-      dockerfile: ./docker/Dockerfile
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+      context: ../ui/
+      dockerfile: ./docker/Dockerfile.gradio
```

Comment:
Here we change the content of a published docker image to something complete different, svelte UI -> gradio UI, without using a new image name. I'm wondering this is the right way to do.

Reply:
Yes, this is not allowed. Do not delete other images. Just create a new one.